test/unit/resources/mysql_session_test.rb

```diff
@@ -17,4 +17,13 @@
     _(resource.send(:create_mysql_cmd, "SELECT 1 FROM DUAL;"))
       .must_equal('mysql -h localhost -s -e "SELECT 1 FROM DUAL;"')
   end
+  it "verify mysql_session correctly redacts sensitive passwords" do
+    resource = load_resource("mysql_session",
+                             "root",
+                             "MYPASSWORD",
+                             sentitive: true)
```

Comment:
I think this one might pass but just need to catch a quick typo

Reply:
Definitely! Good catch.